stencil-library/src/components/payment-form/test/payment-form.e2e.tsx

```diff
@@ -0,0 +1,33 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('justifi-payment-form', () => {
+  it('should correctly receive and set the props', async () => {
```

Comment:
For the e2e tests I think it probably makes more sense to test how the props affect the UI vs checking that it has the expected props. For example, `submit-button-text="Pay Now"`, should actually cause the button to have that text. For the unit tests maybe we can have one that is something like this:
```
const paymentForm = new PaymentForm();
expect(paymentForm).toHaveProperty('submitButtonText', undefined);
```
Just to make sure that the properties exist, so that if they are ever removed from the component we are aware of the breaking change?

Reply:
Yes, totally makes sense, thanks for the suggestion.